doc/sphinx-guides/source/admin/big-data-administration.rst

```diff
@@ -0,0 +1,312 @@
+  all parts of a file cannot be transferred, orphaned files or parts of files can be left on S3. These files are not accessible via Dataverse but do use space (for which there is a monetary cost)
+  until they are deleted. There is currently no automated clean-up mechanism.
+
+Other Considerations
+^^^^^^^^^^^^^^^^^^^^
+
+- S3 Storage without direct upload/download provides minimal benefits with Dataverse as files still pass through the server, files are still uploaded as a single HTTP/HTTPS stream, and temporary storage is still used.
+- While not having files unzipped can be confusing to users who are used to it from using Dataverse with file storage, there are ways to minimize the impact. 
+  For example, Dataverse can be configured to use a 'Zip File Previewer' that allows users to see the contents of a zip file and even download individual files from within it. 
+  For users who still want their data stored as individual files with their relative folder paths, Dataverse can be configured with the 'DVWebloader' which allows users to select an entire folder tree of files and 
+  upload them, with their relative paths intact, to Dataverse. (DVWebloader can only be used with S3/direct upload, but it is much more efficient with many files than using the 
+  standard upload interface in Dataverse (which also does not retain path information)).
+- The following features are disabled when S3 direct upload is enabled.
+  - Unzipping of zip files. (See :ref:`compressed-files`.)
+  - Detection of file type based on JHOVE and custom code that reads the first few bytes except for the refinement of Stata file types to include the version. (See :ref:`redetect-file-type`.)
+  - Extraction of metadata from FITS files. (See :ref:`fits`.)
+  - Creation of NcML auxiliary files (See :ref:`netcdf-and-hdf5`.)
+  - Extraction of a geospatial bounding box from NetCDF and HDF5 files (see :ref:`netcdf-and-hdf5`) unless :ref:`dataverse.netcdf.geo-extract-s3-direct-upload` is set to true.
+
+- Using direct upload stops Dataverse from inspecting the file bytes to determine the MIME type (with one exception - Stata files). Dataverse will still look at the file name and extension to determine the MIME type.
+- To perform the 'ingest' processing, Dataverse currently has to copy the file to local storage, somewhat negating the benefit of sending data directly to S3. To manage larger files, one can set a per-store
+  ingest size limit (which can be 0 bytes) to stop ingest or limit it to smaller files. 
+- Dataverse's mechanism for downloading a whole dataset or multiple selected files involves zipping those files together. Even When using S3 with direct upload/download,
+  the file bytes are transferred to the Dataverse server as part of the zipping process. There are ways to reduce the performance impact of this:
+  - There is a 'ZipDownloader' app that can be run separate from Dataverse to handle the zipping process.
```

Comment:
```suggestion
  - There is a "ZipDownloader" app that can be run separate from Dataverse to handle the zipping process.
```

Let's link to it. I assume you're talking about this: https://guides.dataverse.org/en/6.8/installation/advanced.html#standalone-zipper-service-tool